perceval/components/port.py

```diff
@@ -81,6 +87,24 @@ def encoding(self):
     def is_output_photonic_mode_closed(self):
         return False
 
+    @staticmethod
+    def has_logical_state_equivalent() -> bool:
+        return True
+
+    def to_logic_state(self, state : bool) -> LogicalState:
```

Comment:
I think this method does somewhat the opposite of what it says: why does `state` = 0 outputs |0,1>_Logical? Doesn't make sense to me.

Also, I don't like the signature: why is `state` a boolean? It's not explained in the docstring.

Also, the term _logical_ is used everywhere else. Why _logic_ here?